### ISSUE TITLE
Periodic Table does not have an element starting with "J" or "Q" or "W"

### ISSUE THREAD

**Issue by styfle** [nodejs/Release#304]
> All LTS releases will be assigned a "codename" drawn from the names of elements on the Periodic Table of Elements. For each upcoming LTS release, the Release working group will select a handful of candidate names and submit those for a collaborator vote.

I looked up the [List of chemical elements](https://en.wikipedia.org/wiki/List_of_chemical_elements) and noticed there are no elements starting with "J" or "Q" or "W".

I realize that we won't hit "J" for 7 more LTS releases but I'm curious...how will the "codename" be decided in those 3 cases?

------

*UPDATE*: looks like I'm not the first to [notice](https://github.com/nodejs/Release/issues/163#issuecomment-288850282) this problem.
Maybe other [languages](https://github.com/nodejs/Release/issues/163#issuecomment-288882542) could solve it.

**Comment by addaleax** [comment]
It’s not exactly an answer to this question, but you may want to take a look at what we’re discussing for Node 10.x/the letter D (https://github.com/nodejs/Release/issues/291).

(I guess the real answer is “we don’t know”.)

**Comment by mhdawson** [comment]
I think we can just skip those letters.

**Comment by bnb** [comment]
@MylesBorins agreed. I've already had a half dozen people ask me "well, what are you going to do for the letters that don't have a corresponding element" and I don't foresee skipping letters being a simple explanation to them - let alone simple to understand for those that don't ask.

**Comment by styfle** [comment]
I too think that skipping a letter is not the correct solution. Users who are not aware of the full list of code names (think sysadmin who doesn't use node.js but is tasked with updating it) could get confused. You no longer can rely on the alphabet to know how many LTS releases you are behind the current.

It's like the caveat you'll constantly have to explain:

- Why aren't there 26 LTS releases yet we are changing themes?
- Did I skip an LTS? Where's the release after `Iron`?

More people have the alphabet memorized than the periodic table of elements memorized. So if no element is discovered by the time we get to `J`, I say just pick an arbitrary `J` name that sounds like an element, maybe a molecule 🤷‍♂️

**Comment by styfle** [comment]
Let's approve #318 and revisit this in 5 years 😄 👍 5️⃣ 🤓

**Comment by targos** [comment]
https://github.com/nodejs/Release/pull/318 landed

**Comment by styfle** [comment]
@targos Let's reopen this issue since #318 didn't solve the original issue and its almost been 5 years (ok, only 3.5 but gotta get the ball rolling 🙂)

We are going to reach "J" in a couple releases and we don't have a codename for it yet.

cc @Trott @drewfish

**Comment by styfle** [comment]
I vote for [Jabronium](https://www.lexico.com/en/definition/jabroni) 😆

**Comment by jasnell** [comment]
My vote is on [Jumbonium](https://en.m.wikipedia.org/wiki/List_of_fictional_elements,_materials,_isotopes_and_subatomic_particles)

**Comment by bnb** [comment]
+1 to Jumbonium

**Comment by devsnek** [comment]
Javaium

**Comment by richardlau** [comment]
[Jod](https://en.wikipedia.org/wiki/Iodine) which is another name for Iodine in many non-English languages and how it appeared on early period tables.

**Comment by ThePrez** [comment]
Jermanium

**Comment by matthewmayer** [comment]
Q should be Quark

**Comment by matthewmayer** [comment]
Given there are already several chemical elements with names based on planets such as uranium, neptunium, plutonium, a hypothetical chemical element element named after Jupiter would probably be called Jovium.

**Comment by BethGriggs** [comment]
We should probably do something about this soon for J. Maybe we could create a list of proposed options, and then have a small poll? Poll participants TBD. But, I personally would not object to it being public/end user-facing poll providing Release (or TSC) had no objections to any of the proposed options.

**Comment by Ethan-Arrowood** [comment]
@BethGriggs wdyt about having this discussion / preliminary vote at OpenJS World? I think it could be a fun activity to add to the mix 😄

**Comment by bnb** [comment]
> Poll participants TBD. But, I personally would not object to it being public/end user-facing poll providing Release (or TSC) had no objections to any of the proposed options.

This might be a good use of GitHub Discussions new [Poll feature](https://github.blog/2022-04-12-whats-new-in-github-discussions-organization-discussions-polls-and-more/). Create a poll with pre-agreed options  (probably TSC or collaborators being the ones to choose them) in nodejs/node and share it as widely as possible.

**Comment by MylesBorins** [comment]
From: https://www.thoughtco.com/what-letter-is-not-found-in-the-periodic-table-606637#:~:text=The%20letter%20%22J%22%20is%20the,known%20by%20the%20name%20jod.

> The letter "J" is the only one not found [on the periodic table](https://www.thoughtco.com/clickable-periodic-table-of-the-elements-3891282).


In some countries (e.g., Norway, Poland, Sweden, Serbia, Croatia), the element [iodine](https://www.thoughtco.com/iodine-facts-606546) is known by the name jod.

**Comment by BethGriggs** [comment]
We reviewed all of the potential names suggested in this issue for "J"  in the recent Release WG meeting (https://github.com/nodejs/Release/issues/758, 6 attendees) and the outcome was:
>   * Reviewed the candidate names for “J” and the only option those present in the meeting felt was suitable was “Jod”.

So, a poll is likely not necessary for "J" after all. We agreed to review in the next meeting to allow time for those not present to object (or indicate their support for one of the alternatives). If neither of those happen, we'll PR "Jod" to `CODENAMES` after the next meeting.

**Comment by hashimoto-AJ** [comment]
Japanium😁
https://mazinger.fandom.com/wiki/Japanium

**Comment by styfle** [comment]
Unfortunately, [Jabronium](https://github.com/nodejs/Release/issues/304#issuecomment-999195580) didn't make the cut 😄 

Looks like the PR for [Jod](https://github.com/nodejs/Release/issues/304#issuecomment-999567971) was merged last year and I missed it:

- https://github.com/nodejs/Release/pull/768

Now we get to move on to bikeshedding "Q"...

**Comment by styfle** [comment]
I'm full of ideas:

- Quartz
- Quark
- Quine
- Quicksilver
- Quinoa
- Quahog

**Comment by matthewmayer** [comment]
There is only one option for K (Krypton) so you could go ahead and add that for 2025

**Comment by ljharb** [comment]
also W is the symbol for Tungsten, so that one seems easy

**Comment by matthewmayer** [comment]
Wungsten

**Comment by matthewmayer** [comment]
W for Tungsten comes from Wolfram which would be a good name except for the possible confusion with the language/company/scientist

**Comment by hashimoto-AJ** [comment]
In the first place, when you derived the code name from the chemical element, did you not consider the 10th, 17th, and 23rd concerns?🤔

**Comment by ocmstone** [comment]
So, check this out [Element Names in Different Languages](https://sciencenotes.org/element-names-different-languages/)

I see one option for "Q".  Mercury in German is Quecksilber (translated to English is Quicksilver, which has been used historically).  And it sounds cool!!!  That's my vote for 2031

**Comment by ocmstone** [comment]
And there's only one good option starting with "W", which is Wolfram (or German for Tungsten).
